### PR TITLE
genericx86-64-ext: Add a genericx86-64 extended device type

### PR DESCRIPTION
**Pull request by alexgg**
This new device type is based on the genericx86-64 machine family but
extends the support to be as wide as possible.

Change-type: patch
Changelog-entry: Add genericx86-64-ext device type
Signed-off-by: Alex Gonzalez <alexg@balena.io>

### CODE REVIEW

**Comment by floion** [comment]
@balena-ci retest

File: genericx86-64-ext.coffee [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+	GENERIC_POWERON
+]
+
+module.exports =
+	version: 1
+	slug: 'genericx86-64-ext'
+	aliases: [ 'genericx86-64-ext' ]
+	name: 'Generic x86_64'
+	arch: 'amd64'
+	state: 'experimental'
```
Comment (floion) [inline genericx86-64-ext.coffee:25]: Let's put this straight to `released`. Or if not, to `new`. We do not use `experimental` anymore.

Comment (alexgg) [reply genericx86-64-ext.coffee]: Fixe, thanks.

**Comment by floion** [comment]
@balena-ci retest

**Pull request merged**